<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccWindow</class>
 <widget class="QMainWindow" name="AccWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_self">
         <property name="text">
          <string>self</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cail">
         <property name="text">
          <string>cail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Acc">
         <property name="text">
          <string>Acc</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_in_m1">
      <property name="text">
       <string>IN_M1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_in_m2">
      <property name="text">
       <string>IN_M2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_heart">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_statue">
      <property name="text">
       <string>sttue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_tune">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
